scripts/release_info.py

```diff
@@ -243,14 +245,47 @@ def main():
             for item in storage_changes:
                 out.write('* %s  \n' % item)
 
-        out.write('\n### Authors:\n')
+        past_authors = {
+            log.email: log.author for log in past_logs
+        }
+        release_authors = {(log.author, log.email) for log in new_release_logs}
+
+        new_authors = sorted(set(
+            [(name, email) for name, email in release_authors
+             if email not in past_authors]))
+        existing_authors = sorted(set(
+            [(name, email) for name, email in release_authors
+             if email in past_authors]))
+        new_author_names = [name for name, _ in new_authors]
+        existing_author_names = [name for name, _ in existing_authors]
```

Comment:
As a heads-up, a brief manual sanity-check of the generated output by the release coordinator might still be necessary. Sometimes the same person uses multiple names (often when they forget to configure their git username/email in the first PR they do).

As a side note, we might also want to suggest that release coordinators for the next two releases manually review the output generated by this script just in case there are issues (like the ones we just found due to authors being listed incorrectly).

Reply:
I think the release coordinator will need to review the output, anyway, since it doesn't properly extract names from the user's GitHub profile (or sometimes names aren't even available).

Reply:
OK. May want to update the release doc accordingly.

Reply:
Done (see section 2.7).